native/cocos/editor-support/spine/Vector.h

```diff
@@ -72,14 +75,14 @@ class SP_API Vector : public SpineObject {
         return _size;
     }
 
-    inline void setSize(size_t newSize, const T &defaultValue) {
+    void setSize(size_t newSize, const T &defaultValue) {
```

Comment:
Remove `inline` keyword of function that contains many lines. Let compiler deside whether it should be inlined.